src/app/component/shared/header/header.component.html

```diff
@@ -62,9 +64,20 @@
 
       <!-- Right aligned authenticated links -->
       <ul class="navbar-nav" *ngIf="user">
+        <li *ngIf="user.isAdmin" class="nav-item">
```

Comment:
this could use predicate?

Reply:
Probably, however the `isAdminPredicate` is not functional currently. The issue for it is here: #144

Reply:
Hmm, I dislike how SessionUser is such a weak version of User.

I always just passed a `User` model to menu predicates. I'm not sure why the session user is being passed.